src/main/java/seedu/address/MainApp.java

```diff
@@ -191,7 +194,8 @@ public void start(Stage primaryStage) {
 
     @Override
     public void stop() {
-        logger.info("============================ [ Stopping Address Book ] =============================");
+        logger.info("============================ [ Stopping Lesson and Student Books ] ========================"
```

Comment:
In this case, would it be better to just call it TeachWhat?
```        logger.info("============================ [ Stopping TeachWhat! ] ========================"```

Reply:
Yep